views/home.njk

```diff
@@ -7,34 +7,36 @@
       <div class="usa-grid">
         <div class="usa-hero-callout usa-section-dark">
           <h1>
-            Federalist compliantly hosts federal government websites.
+            Launch a site in 30 minutes. 
```

Comment:
Per https://docs.google.com/document/d/1Sf5ie1eUqsiqQbZAjgpFPfnbr5ZfTXWSGqm4QktOsG4/edit# - I don't know if the launch speed is the key need for people. Federalist is *technically capable* of it but the compliance is more important for folks, I think.

Reply:
From that document it seems like ease of use and speed are key factors.  I think the compliance is a feature, but I don't know if it should be the primary focus of the lead messaging?

Reply:
Maybe split the difference? Something like "Launch a compliant site in 30 minutes."

Reply:
I think _speed of updates_ is more important than _speed of launch_. The other stat might be that it only takes three weeks to do the paperwork and tech work to launch a site - the technical launch speed by itself is less important.

Reply:
Made some edits here to enforce ease of us (in addition to adding compliance).